src/sql/parts/modelComponents/tree/treeComponent.css

```diff
@@ -2,7 +2,12 @@
 	display: flex;
 }
 
-.tree-component-node-tile .model-view-tree-node-item-icon{
-	width: 15px;
+.tree-component-node-tile .model-view-tree-node-item-icon {
+	width: 17px;
 	height: 15px;
```

Comment:
is the image you are using svg? I think SVG will adjust it self.

Reply:
I found icon image set by 16px x 16px slightly miss aligned.
17px x 17px makes perfect alignment.

![image](https://user-images.githubusercontent.com/19577035/56161030-6fa2c600-5f7d-11e9-9118-e38ee70fa2cc.png)

Reply:
`.model-view-tree-node-item-icon` class currently used only in Virtualize Data Extension.